src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -398,19 +374,19 @@ export function ConfirmationModal ({ transaction, handleClose, declineTransactio
       contentLabel = 'Pay request'
       actionParams = { id, amount, counterparty }
       actionHook = payTransaction
-      message = <div styleName='modal-text' data-testid='modal-message'>
```

Comment:
I don't see any updates to the tests that would adjust to this change.  Are not we verifying the message is correct in the tests?

Reply:
This doesn't change the message, it just moves the styling and data id out of this variable and into the main component jsx